google-cloud-spanner/src/test/java/com/google/cloud/spanner/connection/RandomResultSetGenerator.java

```diff
@@ -41,6 +42,7 @@
         Type.newBuilder().setCode(TypeCode.BOOL).build(),
         Type.newBuilder().setCode(TypeCode.INT64).build(),
         Type.newBuilder().setCode(TypeCode.FLOAT64).build(),
+        //        Type.newBuilder().setCode(TypeCode.NUMERIC).build(),
```

Comment:
nit: Commented out code. Maybe we forgot to remove it?

Reply:
Ah, good catch. This is not supposed be commented out at all.